docker-compose.gpu.yml

```diff
@@ -6,6 +6,11 @@ services:
       service: wyoming-piper
     build:
       dockerfile: GPU.Dockerfile
+    ports:
```

Comment:
Are the ports also not extended? I didn't seem to have any problems with it, I need to further investigate. Maybe there is a bug for your docker version. I'm running Ubuntu 23.04 on Docker 24.0.6.